packages/forms-web-app/__tests__/unit/routes/before-you-start/local-planning-department.test.js

```diff
@@ -0,0 +1,31 @@
+const { get, post } = require('../router-mock');
+const localPlanningDepartmentController = require('../../../../src/controllers/before-you-start/local-planning-department');
+const { validationErrorHandler } = require('../../../../src/validators/validation-error-handler');
+const fetchExistingAppealMiddleware = require('../../../../src/middleware/fetch-existing-appeal');
+const {
+  rules: localPlanningDepartmentValidationRules,
+} = require('../../../../src/validators/before-you-start/local-planning-department');
+
+jest.mock('../../../../src/validators/before-you-start/local-planning-department');
+
+describe('routes/eligibility/local-planning-department', () => {
```

Comment:
Incorrect path is referred here
```suggestion
describe('routes/before-you-start/local-planning-department', () => {
```

Reply:
Edited.